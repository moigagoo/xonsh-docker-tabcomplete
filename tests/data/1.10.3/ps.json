{
        "arguments": [
                "-a", "--all",
                "-f", "--filter=[]",
                "--format",
                "--help",
                "-l", "--latest",
                "-n=-1",
                "--no-trunc",
                "-q", "--quiet",
                "-s", "--size"
        ],
        "commands": []
}
