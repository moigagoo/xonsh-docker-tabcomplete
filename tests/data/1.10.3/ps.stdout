
Usage:	docker ps [OPTIONS]

List containers

  -a, --all          Show all containers (default shows just running)
  -f, --filter=[]    Filter output based on conditions provided
  --format           Pretty-print containers using a Go template
  --help             Print usage
  -l, --latest       Show the latest created container (includes all states)
  -n=-1              Show n last created containers (includes all states)
  --no-trunc         Don't truncate output
  -q, --quiet        Only display numeric IDs
  -s, --size         Display total file sizes
